### ISSUE TITLE
Unable to login with existing 5.16 wallet

### ISSUE THREAD

**Issue by ShawnLeary** [NewEconomyMovement/NemCommunityClient#304]
running on Ubuntu.

I had been running 5.16 for the past day with no issues.  I had created a new wallet and named it Ubuntu 5.16

I used "Add an Existing Account" to bring in the 40M XEM I still had from gimre token.  I also set it as Primary Account.

This morning I took everything offline and went into Settings and changed my AutoBoot settings renaming my server name with [colors] so I will look cool on nembex.

Everything restarts normally in both terminal windows.  It takes sometime for NIS to accept the entire blockchain.  When I go to login I see my wallet there, but after putting in the password I am unable to press the OPEN button.  It remains grayed out and non-functional.  The only way I can start is if I create a new wallet.

**Comment by gimer** [comment]
will investigate, but there's a chance it's due to "special" characters in wallet name

**Comment by ShawnLeary** [comment]
good point.  i have another wallet though that didn't have the period, can't remember if it was a 5.5 wallet or 5.13, and it wouldn't let me click OPEN either.  I'll keep playing with it.  Thanks.

**Comment by fyahfox** [comment]
it is known that old wallets led to problems with newer versions.

@wallet name: please suppress "special" charactersin the GUI then (in the textbox where a user can set the name).
